Testing/functional/tests/cypress/integration/e2e/user/vaccineCardQuickLink.js

```diff
@@ -0,0 +1,123 @@
+const { AuthMethod } = require("../../../support/constants");
+
+const homePath = "/home";
+
+const encounterModule = "Encounter";
+const immunizationModule = "Immunization";
+const laboratoryModule = "Laboratory";
+const allLaboratoryModule = "AllLaboratory";
+const medicationModule = "Medication";
+const medicationRequestModule = "MedicationRequest";
+const noteModule = "Note";
```

Comment:
Nit: Looks like constant 'noteModule' is used only once.  Is it necessary to have a constant?